Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateAVFoundationObjC.mm

```diff
@@ -4092,6 +4094,10 @@ void determineChangedTracksFromNewTracksAndOldItems(MediaSelectionGroupAVFObjC*
         });
         ALWAYS_LOG(LOGIDENTIFIER, "Setting spatialAudioExperience: ", spatialAudioExperienceDescription(experience.get()));
         [m_avPlayer setIntendedSpatialAudioExperience:experience.get()];
+
+        if (RefPtr internalMessageClient = player->protectedMessageClientForTesting())
```

Comment:
Nit: I've learned that this is an incorrect use of the `protected` pattern. Apparently in cases where we store to a local variable we should call a `messageClientForTesting()` function that returns a `MessageClientForTesting*`.

(Which begs the question: when would you ever have a `protectedFoo()` that returns `RefPtr`, since you always want to null-check those)